Ark.Tools.ResourceWatcher/ResourceWatcher.cs

```diff
@@ -188,14 +188,16 @@ protected virtual async Task _runOnce(RunType runType, CancellationToken ctk = d
 
         private List<ProcessContext> _createEvalueteList(List<IResourceMetadata> list, IEnumerable<ResourceState> states)
         {
+            Tuple<string, LocalDateTime?, LocalDateTime?> a;
+
             var ev = list.GroupJoin(states, i => i.ResourceId, s => s.ResourceId, (i, s) =>
              {
                  var x = new ProcessContext { CurrentInfo = i, LastState = s.SingleOrDefault() };
                  if (x.LastState == null)
                  {
                      x.ProcessType = ProcessType.New;
                  }
-                 else if (x.LastState.RetryCount == 0 && _getModified(x.CurrentInfo) > _getModified(x.LastState))
+                 else if (x.LastState.RetryCount == 0 && x.IsNewResource(out a))

```

Comment:
when you don't need an out variable, use x.IsNewResource(out _)

https://docs.microsoft.com/en-us/dotnet/csharp/discards#calls-to-methods-with-out-parameters